src/uniref/config/__tests__/__snapshots__/UniRefColumnConfiguration.spec.tsx.snap

```diff
@@ -1,20 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`UniRefColumnConfiguration component should render column "common_taxon": common_taxon 1`] = `
-<DocumentFragment>
-  Xenopus tropicalis (Western clawed frog) (Silurana tropicalis)
-</DocumentFragment>
-`;
+exports[`UniRefColumnConfiguration component should render column "common_taxon": common_taxon 1`] = `<DocumentFragment />`;
```

Comment:
Not intentional, I'll look into that today, I need to update the mock data.
(We badly need to enforce TS type checking on test files too!!! Otherwise our mocks get out of sync with the schema)

Reply:
OK, updated now